src/components/TrendChip.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <app-chip
+    :variant="priceChipVariant">
+    {{ priceChangeSign }}{{ formatNullable(delta) }} %
+  </app-chip>
+</template>
+<script setup>
+const props = defineProps({
+  delta: {
+    type: Number,
+    required: true,
+  },
+})
+
+const priceChangeSign = computed(() => {
+  return props.delta > 0 ? '+' : ''
+})
+const priceChipVariant = computed(() => props.delta > 0 ? 'success' : 'error')
```

Comment:
```suggestion
const priceChipVariant = computed(() => props.delta => 0 ? 'success' : 'error')
```

I would display 0.0 change as success, rather than error.